hnn_core/pyramidal.py

```diff
@@ -372,10 +402,8 @@ def topol(self):
         self.dends['basal_2'].connect(self.dends['basal_1'], 1, 0)
         self.dends['basal_3'].connect(self.dends['basal_1'], 1, 0)
 
-    def _biophys_soma(self):
+    def _biophysics(self):
```

Comment:
this makes it consistent with basket cells. Also I think it's just nicer to have all the biophysics defined in one place.